pyproject.toml

```diff
@@ -21,15 +21,16 @@ classifiers = [
 ]
 keywords = ["lsst"]
 dependencies = [
-    "astropy >= 4",
-    "sqlalchemy >= 1.4",
-    "click >= 7",
-    "pyyaml >= 6",
-    "pydantic >= 2, < 3",
+    "astropy >= 6.0.0",
+    "numpy < 2",
+    "sqlalchemy >= >=2.0.23",
```

Comment:
This seems wrong.

Reply:
Which dependency seems wrong here?

Reply:
Oh, I see. The `sqlalchemy` version needs to be updated to `>=2.0.23`. Will do...